src/lib/pages/backoffice/Actions/CheckIn/ItemsSearch/ItemsSearch.js

```diff
@@ -50,11 +73,15 @@ export default class ItemsSearch extends Component {
           currentQueryString={queryString}
           onInputChange={this.onInputChange}
           executeSearch={this.onKeyPressHandler}
-          placeholder="Scan physical copy barcode to check-in..."
+          placeholder={
+            isCheckout
+              ? 'Scan physical copy barcode to checkout...'
```

Comment:
```suggestion
              ? 'Insert physical copy barcode to checkout...'
```
and also below. The thing is that we don't know if librarian will really scan, it depends on the libraries equipment.
It was already there, we missed it... can you please make sure there is no "scan" mentioning around?